app/lib/custom_log_formatter.rb

```diff
@@ -0,0 +1,76 @@
+require 'semantic_logger'
+
+class CustomLogFormatter < SemanticLogger::Formatters::Raw
+  REDACTED = '[REDACTED]'.freeze
+
+  def call(log, logger)
+    super
+
+    add_custom_fields
+    sanitize_payload_fields
+    remove_post_params
+    filter_skipping_email_message
+
+    # rubocop:disable Rails/Output
+    puts '-' * 80
+    puts "Hash: #{hash}"
+    puts '-' * 80
```

Comment:
This is only here for check that this class is being called. Which when performed the delivery of the email this line is not being called so the custom log is not used when Sidekiq finishes or skip the delivery of the email